### PR TITLE
Features/docker image

### PR DESCRIPTION
**Pull request by kiselev-dv**
Pack nodes into Docker container

### CODE REVIEW

**Comment by kiselev-dv** [comment]
test it please

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    tar \
+    libfreeimage3 \
+    xdg-utils \
+    onboard \
+    git \
+    libexif12 \
+    libexif-dev \
+    vim \
+    nano \
+    mc
```
Comment (minshallj) [inline docker/Dockerfile]: you probably want a `&& rm -rf /var/lib/apt/lists/*` here, as each `RUN` command marks a commit when building a docker image.

**Comment by wzin** [comment]
test ros

**Comment by wzin** [comment]
test ros

**Pull request closed without merging**